pxr/usd/usdShade/testenv/testUsdShadeValidators.cpp

```diff
@@ -23,40 +23,75 @@
 //
 
 #include "pxr/pxr.h"
-#include "pxr/usd/usd/validationRegistry.h"
+
+#include "pxr/base/tf/token.h"
+#include "pxr/usd/sdf/layer.h"
+#include "pxr/usd/sdf/path.h"
+#include "pxr/usd/usd/prim.h"
+#include "pxr/usd/usd/stage.h"
 #include "pxr/usd/usd/validationError.h"
+#include "pxr/usd/usd/validationRegistry.h"
 #include "pxr/usd/usd/validator.h"
-
+#include "pxr/usd/usdGeom/validatorTokens.h"
 #include "pxr/usd/usdShade/validatorTokens.h"
 
 #include <algorithm>
+#include <set>
+#include <string>
 
 PXR_NAMESPACE_USING_DIRECTIVE
 
 void
 TestUsdShadeValidators()
 {
-    // This should be updated with every new validators added with
+    // This should be updated with every new validator added with the
     // UsdShadeValidators keyword.
-    UsdValidationRegistry &registry = UsdValidationRegistry::GetInstance();
-    UsdValidatorMetadataVector metadata = 
+    const std::set<TfToken> expectedUsdShadeValidatorNames = {
```

Comment:
I ended up re-organizing this a little bit since we're interested in both all of the validators for the `UsdShadeValidators` keyword *and* the `UsdGeomSubset` keyword. The latter is a keyword we re-use from `usdGeom`'s validator keyword tokens.